changelog.d/20231108_133040_faraz.maqsood_update_the_version.md

```diff
@@ -0,0 +1 @@
+- [BugFix] Version update that includes Correction of variable name for installing extra pip requirements. (by @Faraz32123)
```

Comment:
You should not a changelog entry when you release a new version. Instead, run `scriv collect` to aggregate all changelog entries into CHANGELOG.md.